docs/builds.md

```diff
@@ -35,5 +35,6 @@ containers.
 
 The following git repositories have some useful examples:
 
+- [centos-boot-examples](https://gitlab.com/CentOS/cloud/centos-boot-examples)
```

Comment:
The linked repository is using the old project name.

I don't feel strongly about it but would prefer if the examples would be integrated into the repository here.  Since the examples use centos, the GitHub repo feels like a good home.

Reply:
we can merge this as/is but make an issue to move all examples here - that way they're easy to find now until @cgwalters has time to move them here.

Reply:
i agree condensing the repos to as few as possible is a good thing

Reply:
Yeah...I know...I guess the thing is in my mind the git*hub* move was temporary because RHTAP needed it.  But clearly it's already becoming entrenched.  (background: A lot of RHEL stuff is on https://gitlab.com/redhat/ including Stream...since the examples don't need to be built from RHTAP I was thinking we could keep stuff there.  But yes the specific gitlab hosting is old)